### PR TITLE
Project development

### PR DESCRIPTION
**Pull request by danidreis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <title>Dani's Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <nav>
+        <img src="Images/My-Logo.png" alt="Logo Symbol"/>
+    </nav>
```
Comment (koachcode) [inline index.html]: - Please `<header>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            </p>     
+            <h3>Let's connect</h3>
+        </div>
+        <div class="social">
+            <img class="button" src="Images/Vector-3.svg" alt="Twitter Icon"/>
+            <img class="button" src="Images/Linkedin-icon.svg" alt="Linkedin Icon"/>
+            <img class="button" src="Images/Vector-1.svg" alt="Gmail Icon"/>
+            <img class="button" src="Images/Vector-2.svg" alt="Cat Icon"/>
+            <img class="button" src="Images/Vector.svg" alt="Peace Icon"/>
+        </div>
```
Comment (koachcode) [inline index.html]: - Please make use of `<ul>` tag with the corresponding list tag `<li>` so as to properly align the icons and make your code semantically correct

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+### Prerequisites
+- You will need a working computer and internet conection.
+
+## Authors
+
+👤 **Daniela Reis**
+
+- GitHub: [@danidreis ](https://github.com/danidreis)
+- Twitter: [@DevDanielaReis](https://twitter.com/DevDanielaReis)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/daniela-reis-469a4a23b/)
```
Comment (koachcode) [inline README.md]: - Please include the MIT lincense. Use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a reference.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @danidreis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

you-are-near

## Highlights
- Linters all green 💯 
- Professional commit message 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <title>Dani's Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <img src="Images/My-Logo.png" alt="Logo Symbol"/>
+    </header>
```
Comment (Kakalanp) [inline index.html:16]: [OPTIONAL]

Remember that, when the time comes to add a hamburger button, you should do it in a `<nav>` tag to follow best practices :wink:

File: README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (Kakalanp) [inline README.md:3]: [OPTIONAL]

For future projects, never forget to update your project's title in your README :sweat_smile: it's sometimes the best way to show it off :+1:

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @danidreis! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**